presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/TestSqlServerIntegrationSmokeTest.java

```diff
@@ -111,6 +111,18 @@ public void testDecimalPredicatePushdown()
         }
     }
 
+    @Test
+    public void testInsertVarbinary()
```

Comment:
This test seems to be passing even without changes in SqlServerClient. 
Intentional?

Reply:
@findepi Sorry, it's unintentional. I should have separated INSERT statements.